action/Run-Maester.ps1

```diff
@@ -40,7 +40,16 @@ param (
     [bool]$DisableTelemetry = $false,
 
     [Parameter(Mandatory = $false, HelpMessage = 'Add test results to GitHub step summary')]
-    [bool]$GitHubStepSummary = $false
+    [bool]$GitHubStepSummary = $false,
+
+    [Parameter(Mandatory = $false, HelpMessage = 'Teams Webhook Uri to send test results to, see: https://maester.dev/docs/monitoring/teams')]
+    [string]$TeamsWebhookUri = $null,
+
+    [Parameter(Mandatory = $false, HelpMessage = 'Teams notification channel ID')]
+    [string]$TeamsChannelId = $null,
+
+    [Parameter(Mandatory = $false, HelpMessage = 'Teams notification teams ID')]
```

Comment:
Super nit pick: HelpMessage should be 'Teams notification **team** ID'. :)

Reply:
Good find!